client/src/templates/Challenges/classic/editor.tsx

```diff
@@ -894,6 +898,27 @@ const Editor = (props: EditorProps): JSX.Element => {
     updateFile({ fileKey, editorValue, editableRegionBoundaries });
   };
 
+  function createBreadcrumb(): HTMLOListElement {
+    const { block, superBlock } = props;
+    const breadcrumb = document.createElement('ol');
+    const breadcrumbLeft = document.createElement('li'),
+      breadcrumbLeftLink = document.createElement('a'),
+      breadcrumbRight = document.createElement('li'),
+      breadcrumbRightLink = document.createElement('a');
+    breadcrumbLeftLink.innerHTML = t(`intro:${superBlock}.title`);
+    breadcrumbRightLink.innerHTML = t(
+      `intro:${superBlock}.blocks.${block}.title`
+    );
+    breadcrumbLeftLink.setAttribute('href', `/learn/${superBlock}`);
+    breadcrumbRightLink.setAttribute('href', `/learn/${superBlock}/#${block}`);
+    breadcrumbLeft.appendChild(breadcrumbLeftLink);
+    breadcrumbRight.appendChild(breadcrumbRightLink);
+    breadcrumb.appendChild(breadcrumbLeft);
+    breadcrumb.appendChild(breadcrumbRight);
+
+    return breadcrumb;
+  }
+
```

Comment:
Where can I find this function being used? I don't see it directly in the jaw

Reply:
Actually, this will be called in `createDescription` function and `useEffect` hook which monitors the `isMobileHeight`.

I'm not sure whether this is best approach to control of displaying minimized breadcrumb in the jaw because this seems not to be Reactive, but I couldn't find any other way to make the control responsive with viewport height. Therefore this function was used in the function and hook.